client/src/firebase.ts

```diff
@@ -12,6 +13,12 @@ const firebaseConfig = {
 
 export const app = initializeApp(firebaseConfig);
 export const auth = getAuth(app);
+export const db = getFirestore(app);
 if (process.env.NODE_ENV !== "production") {
-  connectAuthEmulator(auth, "http://localhost:9099");
+  import("firebase/auth").then(({ connectAuthEmulator }) =>
```

Comment:
why move the import to inside the 'if' block?
and it really neccesarry to use ".then()"?
here [source](https://github.com/FirebaseExtended/reactfire/blob/main/docs/use.md)
i found this:
```
import { getAuth, connectAuthEmulator } from 'firebase/auth'; // Firebase v9+
...
  if (process.env.NODE_ENV !== 'production') {
    connectAuthEmulator(auth, 'http://localhost:9099');
  }
```

Reply:
I thought it might help the bundler bundle less code